left_arm:
  P1:
    cx: 20.0
    cy: 404.0
    ik_x: 0.25
    ik_y: 0.28
  P2:
    cx: 71.0
    cy: 167.0
    ik_x: 0.55
    ik_y: 0.28
  P3:
    cx: 285.0
    cy: 147.0
    ik_x: 0.55
    ik_y: 0.0
  P4:
    cx: 281.0
    cy: 363.0
    ik_x: 0.25
    ik_y: 0.0
right_arm:
  P1:
    cx: 618.0
    cy: 419.0
    ik_x: 0.25
    ik_y: -0.28
  P2:
    cx: 569.0
    cy: 184.0
    ik_x: 0.55
    ik_y: -0.28
  P3:
    cx: 349.0
    cy: 149.0
    ik_x: 0.55
    ik_y: -0.0
  P4:
    cx: 344.0
    cy: 367.0
    ik_x: 0.25
    ik_y: -0.0
